.github/config.yml

```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to Meshery! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/layer5io/meshkit/master/.github/welcome/Layer5-celebration.png)
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./welcome/Meshery-celebration-w.png">
+    <source media="(prefers-color-scheme: light)" srcset="./welcome/Meshery-celebration-b.png">
+    <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
+  </picture>
```

Comment:
Thanks for the changes @om2137 🛩️ 

There are some changes required.

- The ```srcset``` needs to be a reference to absolute path to raw image.
- The name of the images needs to be more clear, as explained by @thisiskaransgit 
- Also, I think, you have interchanged the white image to dark color scheme, and black to light color scheme.

Please refer to the following example

```
Thanks for your contribution to Meshery! :tada:
   
  <picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/Meshery-celebration-Dark.svg">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/Meshery-celebration-Light.svg">
  <img alt="Meshery Logo" src="">
  </picture>
```

Or, Following Merged PR, referring to a similar issue - https://github.com/meshery/meshery/pull/7586/files